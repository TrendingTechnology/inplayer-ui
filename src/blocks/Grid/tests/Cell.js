import React from 'react';
import { renderWithTheme } from 'helpers';
import Cell from '../Cell';
import 'jest-styled-components';

describe('Grid.Column Element', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(<Cell />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('applies the correct style with the `flex` prop', () => {
    const tree = renderWithTheme(<Cell />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('applies the correct style with the `position` modifier', () => {
    const tree = renderWithTheme(<Cell />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
