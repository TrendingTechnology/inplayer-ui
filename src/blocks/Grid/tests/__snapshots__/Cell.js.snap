// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid.Column Element applies the correct style with the \`flex\` prop 1`] = `
.c0 {
  height: 100%;
  min-width: 0;
  grid-column-end: span 1;
  grid-row-end: span 1;
}

<div
  className="c0"
/>
`;

exports[`Grid.Column Element applies the correct style with the \`position\` modifier 1`] = `
.c0 {
  height: 100%;
  min-width: 0;
  grid-column-end: span 1;
  grid-row-end: span 1;
}

<div
  className="c0"
/>
`;

exports[`Grid.Column Element renders correctly 1`] = `
.c0 {
  height: 100%;
  min-width: 0;
  grid-column-end: span 1;
  grid-row-end: span 1;
}

<div
  className="c0"
/>
`;
